client/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@monaco-protocol-event/client",
```

Comment:
`@monaco-protocol` is the "org", I think that should be the same across all our packages, then `/` followed by the package name, so 

```
@monaco-protocol/event-client
```
or 
```
@monaco-protocol/protocol-event-client
```